src/combinators/reduction/foldL.ts

```diff
@@ -1,26 +1,52 @@
-import { alt } from "../alternation/mod.ts";
-import { foldL1 } from "./foldL1.ts";
 import type { Parser } from "$core";
+import { fail } from "$core";
+import { many, seq } from "$combinators";
 
 /**
- * Parses maybe-empty sequences of items separated by an operator parser that associates to the left and performs the fold
+ * Repeats an item parser and a left-associative operator parser greedily between min and max times, inclusive
  *
- * @example Addition
+ * @param parser The item parser
+ * @param operator The operator parser, returning a function (a:T, b:T) => T
+ * @param min The minimum number of items to parse
+ * @param max The maximum number of items to parse (default: Infinity)
+ * @returns A parser returning the folded result
  *
- * We lift the addition literal `+` into a binary function parser and apply a left fold
```

Comment:
Why remove the explanation?

Reply:
The important part is the left-associative note, didn't see it's now in the docsting so it's fine.